scripts/core/exp-bindless-images.yml

```diff
@@ -0,0 +1,473 @@
+returns:
+    - $X_RESULT_ERROR_INVALID_CONTEXT
+    - $X_RESULT_ERROR_INVALID_VALUE
+    - $X_RESULT_ERROR_INVALID_IMAGE_FORMAT_DESCRIPTOR:
+      - "`pImageDesc && UR_MEM_TYPE_IMAGE1D_BUFFER < pImageDesc->type`"
+    - $X_RESULT_ERROR_INVALID_IMAGE_SIZE
+    - $X_RESULT_ERROR_INVALID_SAMPLER
+    - $X_RESULT_ERROR_INVALID_OPERATION
+    - $X_RESULT_ERROR_OUT_OF_RESOURCES
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Copy image data Host to Device or Device to Host"
+class: $xBindlessImages
+name: ImageCopyExp
+ordinal: "0"
+analogue:
+    - "**cuMemcpyHtoAAsync**"
+    - "**cuMemcpyAtoHAsync**"
+    - "**cuMemcpy2DAsync**"
+    - "**cuMemcpy3DAsync**"
+params:
+    - type: $x_context_handle_t
+      name: hContext
+      desc: "[in] handle of the context object"
+    - type: void*
+      name: pDst
+      desc: "[in] location the data will be copied to"
+    - type: void*
+      name: pSrc
+      desc: "[in] location the data will be copied from"
+    - type: "const $x_image_format_t*"
+      name: pImageFormat
+      desc: "[in] pointer to image format specification"
+    - type: "const $x_image_desc_t*"
+      name: pImageDesc
+      desc: "[in] pointer to image description"
+    - type: $x_exp_image_copy_flags_t
+      name: imageCopyFlags
+      desc: "[in] flags describing copy direction e.g. H2D or D2H"
+    - type: uint32_t
+      name: numEventsInWaitList
+      desc: "[in] size of the event wait list"
+    - type: "const $x_event_handle_t*"
+      name: phEventWaitList
+      desc: |
+            [in][optional][range(0, numEventsInWaitList)] pointer to a list of events that must be complete before this command can be executed.
+            If nullptr, the numEventsInWaitList must be 0, indicating that all previously enqueued commands
+            must be complete. 
+    - type: $x_event_handle_t*
+      name: phEvent
+      desc: |
+            [out][optional] return an event object that identifies this particular command instance.
+returns:
+    - $X_RESULT_ERROR_INVALID_CONTEXT
+    - $X_RESULT_ERROR_INVALID_VALUE
+    - $X_RESULT_ERROR_INVALID_IMAGE_FORMAT_DESCRIPTOR:
+      - "`pImageDesc && UR_MEM_TYPE_IMAGE1D_BUFFER < pImageDesc->type`"
+    - $X_RESULT_ERROR_INVALID_IMAGE_SIZE
+    - $X_RESULT_ERROR_INVALID_OPERATION
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Query an image memory handle for specific properties"
+class: $xBindlessImages
+name: ImageGetInfoExp
+ordinal: "0"
+params:
+    - type: $x_exp_image_mem_handle_t
+      name: hImageMem
+      desc: "[in] handle to the image memory"
+    - type: $x_image_info_t
+      name: paramInfoName
+      desc: "[in] queried info name"
+    - type: void*
+      name: pParamValue
+      desc: "[out] returned query value"
+    - type: "size_t*"
+      name: pParamValueSize
+      desc: "[out] returned query value size"
+returns:
+    - $X_RESULT_ERROR_INVALID_VALUE
```

Comment:
We have a standard set of return codes for `GetInfo` type queries - you can pretty much copy paste them from [here](https://github.com/oneapi-src/unified-runtime/blob/main/scripts/core/device.yml#L419)

Reply:
Thanks! Done.